src/css/controls/imports/slider.less

```diff
@@ -203,3 +207,9 @@
         }
     }
 }
+
+.jw-breakpoint-0 {
+    .jw-slider-time {
+        height: 11px;
+    }
+}
```

Comment:
I lowered the time slider. For some reason it was really high on the small players and almost touching the play/pause icon.